### PR TITLE
Assign reform_affected column to baseline df to support assignment

### PR DESCRIPTION
**Pull request by hdoupe**
Maybe related to this api change [[docs]](https://pandas.pydata.org/pandas-docs/version/1.1.5/user_guide/indexing.html#selection-by-label)?

image

It's been awhile since pandas 0.20.1 so I'm not sure why this is popping up now.

### CODE REVIEW

File: cs-config/cs_config/helpers.py [hunk 1]
```diff
@@ -120,28 +120,31 @@ def fuzzed(df1, df2, reform_affected, table_row_type):
...
+    df1['reform_affected'] = False
     # construct table rows, for which filing units in each row must be fuzzed
     if table_row_type == 'xbin':
         df1 = add_income_table_row_variable(df1, 'expanded_income',
                                             STANDARD_INCOME_BINS)
         df2['expanded_income_baseline'] = df1['expanded_income']
         df2 = add_income_table_row_variable(df2, 'expanded_income_baseline',
                                             STANDARD_INCOME_BINS)
-        del df2['expanded_income_baseline']
+        df2.drop(columns='expanded_income_baseline', inplace=True)
```
Comment (hdoupe) [inline cs-config/cs_config/helpers.py:132]: Updated the code to drop columns to use more idiomatic pandas. Happy to revert if we want to keep this PR simple.

**Comment by hdoupe** [comment]
@andersonfrailey the `df1` and `df2` variable names are kind of confusing in the `fuzzed` function. I kept getting them mixed up :grimacing:. It might be nice to rename them to `df_base` and `df_reform` in a follow up PR.

**Comment by jdebacker** [comment]
@hdoupe It looks like this PR is intended to address Issue #188.  Is it still relevant?

**Comment by jdebacker** [comment]
Closed due to lack of activity and no issues with Tax-Brain related to this proposed change.

**Pull request closed without merging**